### PR TITLE
Home Page automation tests

### PR DESCRIPTION
**Pull request by lorenacioata**
Here are scripted tests for Home Page.
The task can be found at: https://red.assist.ro/issues/47734

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,14 +26,16 @@
...
     "prettier": "^2.3.2",
+    "set-blocking": "^2.0.0",
     "wdio-chromedriver-service": "^7.2.0"
   },
   "dependencies": {
-    "allure-commandline": "^2.13.8",
-    "set-blocking": "^2.0.0"
+    "chai": "^4.3.4"
   }
 }
```
Comment (TaiaDimitrova) [inline package.json:41]: I'd suggest you to discard all the installed dependencies by running 'git checkout main package.json', then install chai dependency back and push the changes again. This way you'll have only the necessary changes on this branch.

File: test/pageobjects/LoginPage.js [hunk 2]
```diff
@@ -1,7 +1,3 @@
-import Base from '../test/pageobjects/Base.js'
+class LoginPage {}
 
-class LoginPage extends Base {
-    
-}
-
-export default new LoginPage()
\ No newline at end of file
+module.exports = new LoginPage();
```
Comment (TaiaDimitrova) [inline test/pageobjects/LoginPage.js]: I'd suggest removing this file as it does not contain any methods.

File: tests/HomePageSmokeTests.spec.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+const chai = require("chai");
+const expect = chai.expect;
+const App = require("../test/pageobjects/App");
+const HomePage = require("../test/pageobjects/HomePage");
+
+describe("Verifiy page logo", () => {
```
Comment (TaiaDimitrova) [inline tests/HomePageSmokeTests.spec.js:6]: The scenario name should describe all the covered functionalities by the tests, and here we have mentioned only the logo verification.

**Review by TaiaDimitrova — CHANGES_REQUESTED** [review]
Before merging, please, resolve the conflicts.

File: test/pageobjects/LoginPage.js (hunk 2 above)
Comment (lorenacioata) [reply test/pageobjects/LoginPage.js]: Sure.

File: package.json (hunk 1 above)
Comment (lorenacioata) [reply package.json:41]: Can you be more specific about the suggestion please? 
'git checkout main package.json'  gives "Updated 0 paths from 45d8abd" message.

File: tests/HomePageSmokeTests.spec.js (hunk 3 above)
Comment (lorenacioata) [reply tests/HomePageSmokeTests.spec.js:6]: Okay

**Review by TaiaDimitrova — APPROVED** [review]

**Pull request merged**